packages/frontend-vitals-monitoring/README.md

```diff
@@ -0,0 +1,149 @@
+<!-- Improved compatibility of back to top link: See: https://github.com/othneildrew/Best-README-Template/pull/73 -->
+
+<a name="readme-top"></a>
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  
+<h3 align="center">GOV.UK Frontend Vitals Signs</h3>
+  <p align="center">
+    This package enables GOV UK LOGIN frontend Node.js applications add frontend vitals monitoring.
+    <br />
+    <a href=""><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/govuk-one-login/govuk-one-login-frontend/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/govuk-one-login/govuk-one-login-frontend/issues">Request Feature</a>
+  </p>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+     <li><a href="#frontendvitalsinitserver-options">`frontendVitalsInit(server, options)`</a></li>
+  </ol>
+</details>
+
+## About The Project
+
+The GDS Frontend Vitals Signs node package exposes custom metrics (vital-signs) about the health of our Node applications.
+
+The purpose of this package is to make it easy for our FE services to scale in response to demand.
+
+The package is owned by the DI Frontend Capability team, part of the development of this tool involves ongoing discovery with the pods responsible for maintaining the frontend repositories that make up the One Login journey. A stable release of the package is now deployed in production.
```

Comment:
I think maybe leave out `A stable release of the package is now deployed in production.` for now as it might be misinterpreted as meaning that one of our services is using it in production.

Reply:
removed